modules/oe_whitelabel_paragraphs/oe_whitelabel_paragraphs.module

```diff
@@ -272,12 +278,19 @@ function oe_whitelabel_preprocess_paragraph__oe_banner(array &$variables): void
   $variant = $paragraph->get('oe_paragraphs_variant')->value ?? 'default';
   $variables['variant'] = str_replace('oe_banner_', '', $variant);
 
-  if ($variables['variant'] === 'default') {
-    $color_scheme_preprocess->injectColorScheme($variables);
-  }
-  else {
-    $color_scheme_preprocess->injectColorScheme($variables, FALSE);
-  }
+  match($variables['variant']) {
```

Comment:
I had to complicate things here so that the correct color contrasts are applied (for example on the shaded banner, the best text color contrast is with white always)
![Screenshot 2024-06-12 at 16 51 28](https://github.com/openeuropa/oe_whitelabel/assets/11507647/f025910e-b20d-46c4-8dfd-eb531059c7bd)